table.go

```diff
@@ -294,12 +327,12 @@ var TableList = []Table{
 		Schema:              1,
 		Task:                tasktype.VerifiedRegistryVerifiedClient,
 		Model:               &verifreg.VerifiedRegistryVerifiedClient{},
-		NetworkVersionRange: AllNetWorkVersions,
+		NetworkVersionRange: ToNetworkVersion16,
```

Comment:
This task only needs to run up to nv16. It can still be included for jobs covering epochs after network version 17, but will not produce any data in its corresponding model after network version 17.